servicetalk-transport-netty-internal/build.gradle

```diff
@@ -16,6 +16,9 @@
 
 apply plugin: "io.servicetalk.servicetalk-gradle-plugin-internal-library"
 
+ext.isNettySnapshot = "$nettyVersion".endsWithAny("SNAPSHOT", "+")
+println("!!! $nettyVersion")
```

Comment:
is this needed? if so can this be more descriptive (e.g. nettyVersion: ...)?

Reply:
👀 leftover